src/modules.js

```diff
@@ -24,7 +24,7 @@ const filterProperties = (
   .reduce((object, key) => ({ ...object, [key]: properties[key] }), {});
 
 // --- Debug module ---
-if (__analytics.debug?.isEnabled) {
+if (__analytics.debug.isEnabled) {
```

Comment:
so, do all these properties `debug`, `googleTagManager`, `googleAnalytics`, `amplitude` etc have to declared?

Reply:
yes, after this changes it should be, for `?.` we loose tree shaking :(

Reply:
but it minory version :D it should be updated by developer :D npm install will not update it ;) it's works just for PATH

Reply:
Shouldn't debug be disabled by default then?

Reply:
probably it can be :D but you must set it in your project, than it doesn't metr IMO :)

Reply:
I think default should also make sense, I doubt anyone wants debug enabled most of the time. :p

Reply:
sure :) I agree. But when you use modular-analytics you must setup
```
{ 
  debug: {
    isEnabled: true/false
  } 
}
```
You can add issue for that. Fell free to made this package better :) 
But important part of that is tree shaking.